content/docs/user-guide/external-dependencies.md

```diff
@@ -17,20 +17,19 @@ stages. DVC will track changes in them and reflect this in the output of
 Currently, the following types (protocols) of external dependencies are
 supported:
 
-- Local files and directories outside of your <abbr>workspace</abbr>
-- SSH
 - Amazon S3
 - Microsoft Azure Blob Storage
 - Google Cloud Storage
+- SSH
 - HDFS
-- HTTP
```

Comment:
Why are we deleting HTTP though?

```
dvc import-url https://github.com/iterative/dvc/releases/download/1.6.3/dvc-1.6.3-1.x86_64.rpm
```

works fine.

Reply:
I didn't mean to delete that, thanks for noticing! Bad copy/paste, probably 🤦‍♂️